src/stampy_nlp/faq_titles.py

```diff
@@ -81,8 +93,8 @@ def encode_faq_titles():
     logging.basicConfig(level=logging.DEBUG)
 
     if len(sys.argv) > 1 and sys.argv[1] == '-delete-all':
-        logging.debug(f"sys.argv[1]={sys.argv[1]}")
+        logger.debug("sys.argv[1]=%s", sys.argv[1])
         delete_all = True
-    logging.debug(f"delete_all={delete_all}")
+    logger.debug("delete_all=%s", delete_all)
 
```

Comment:
What’s the difference between logger and logging? Is logger generally better or preferred choice for some reason? If there’s a short answer or link to a page to read later, that’d be appreciated.